packages/gosu.rb

```diff
@@ -0,0 +1,27 @@
+require 'package'
+
+class Gosu < Package
+  description 'Simple Go-based setuid+setgid+setgroups+exec'
+  homepage 'https://github.com/tianon/gosu'
+  version '1.14'
+  license 'Apache-2.0'
+  compatibility 'all'
+  source_url 'SKIP'
+
```

Comment:
Add `no_compile_needed` flag?

Reply:
Good call.  Added!

Reply:
Will the `no_compile_needed` flag disable UPX?

Reply:
No, it will only disable `strip`